### PR TITLE
Toggle canonical URLs

### PR DESCRIPTION
**Pull request by svaloumas**

### CODE REVIEW

File: includes/admin/transifex-live-integration-admin-template.php [hunk 1]
```diff
@@ -46,6 +46,11 @@
 				<input name="transifex_live_settings[translate_urls]" type="checkbox" id="transifex_live_settings_translate_urls" value="1" <?php echo $checked_translate_urls ?>><?php _e( 'Make images and links translatable' ); ?>&nbsp;<a target="_blank" href="<?php echo $settings['urls']['translate_urls']; ?>"><b><?php _e( 'Learn more' ) ?></b></a>.
 			</td>
 		</tr>
+		<tr>
+			<td style="padding-right:0px">
+				<input name="transifex_live_settings[canonical_urls]" type="checkbox" id="transifex_live_settings_canonical_urls" value="1" <?php echo $checked_canonical_urls ?>><?php _e( 'Disable rendering canonical URLs' ); ?>&nbsp;<a target="_blank" href="<?php echo $settings['urls']['canonical_urls']; ?>"><b><?php _e( 'Learn more' ) ?></b></a>.
+			</td>
+		</tr>
```
Comment (asgouros-tx) [inline includes/admin/transifex-live-integration-admin-template.php:53]: I think there something wrong with the spacing here (see image). Was it intentional?

File: includes/transifex-live-integration-defaults.php [hunk 2]
```diff
@@ -120,6 +121,7 @@ static function settings() {
 				'rate_us' => 'https://wordpress.org/support/view/plugin-reviews/transifex-live-integration?rate=5#postform',
 				'api_key_landing_page' => 'https://app.transifex.com/signup/?utm_source=liveplugin',
 				'translate_urls' => 'https://help.transifex.com/en/articles/6259014-technical-instructions#h_e9a258ad3b',
+        'canonical_urls' => 'https://developers.google.com/search/docs/crawling-indexing/consolidate-duplicate-urls'
```
Comment (asgouros-tx) [inline includes/transifex-live-integration-defaults.php:124]: Is it acceptable that we point to google docs and not ours? (Maybe it is I am not sure)

File: includes/lib/transifex-live-integration-hreflang.php [hunk 3]
```diff
@@ -164,6 +165,17 @@ public function render_hreflang() {
...
+      $canonical_url = home_url('/');
+      if (!empty($url_path)) {
+          $canonical_url = home_url($url_path);
+      }
+      $canonical_url = rtrim($canonical_url, '/') . '/';
+
+      $hreflang_out .= <<<CANONICAL
+  <link rel="canonical" href="$canonical_url"/>\n
+  CANONICAL;
+    }
```
Comment (asgouros-tx) [inline includes/lib/transifex-live-integration-hreflang.php:178]: As discussed, let's better re-use the way that HREFLANG is constructed as in lines 158-164 above

Comment (svaloumas) [reply includes/lib/transifex-live-integration-hreflang.php]: Updated as discussed!

File: includes/transifex-live-integration-defaults.php (hunk 2 above)
Comment (asgouros-tx) [reply includes/transifex-live-integration-defaults.php:124]: As per discussion with Aggeliki, let's remove for now and we can re-iterate later if needed.

Comment (svaloumas) [reply includes/transifex-live-integration-defaults.php]: Removed!

**Pull request merged**